yaroslav.nazarenko/first/task1/task1.rb

```diff
@@ -0,0 +1,2 @@
+x = Array.new(3){Array.new(3){rand(-10..10)}}
```

Comment:
```suggestion
x = Array.new(3) { Array.new(3) { rand(-10..10) } }
```
It's [better to add spaces](https://github.com/rubocop-hq/ruby-style-guide#spaces-and-operators) here for good readability